src/server/server_family.cc

```diff
@@ -2056,11 +2052,13 @@ Metrics ServerFamily::GetMetrics(Namespace* ns) const {
 
   bool is_master = ServerState::tlocal() && ServerState::tlocal()->is_master;
   if (is_master) {
-    result.replication_metrics = dfly_cmd_->GetReplicasRoleInfo();
+    result.master_side_replicas_info = dfly_cmd_->GetReplicasRoleInfo();
   } else {
     auto info = GetReplicaSummary();
     if (info) {
-      result.replica_reconnections = {std::move(info->host), info->port, info->reconnect_count};
+      result.replica_side_info = {
+          .reconnect_count = info->reconnect_count,
```

Comment:
why do you need designated initializers on one field struct ?

Reply:
I find it more readable so in case more fields are added, there is no confusion what is set with what value

Reply:
Sure just asking, I don't really mind :shrug: